### ISSUE TITLE
Error when using transition property in tailwindcss + emotion example

### ISSUE THREAD

**Issue by phc5** [zeit/next.js#11872]
# Bug report

## Describe the bug

Adding the following classes `transition`, `duration-150`, `ease-in-out` produces this error: 
```
Error: Couldn’t resolve Tailwind class name: transition
e.resolveStyleFromPlugins
/Users/pchong/Desktop/with-tailwindcss-emotion-app/node_modules/tailwind.macro/utils.umd.js:1:2304
Module../pages/index.js
```

## To Reproduce

1. Clone example: https://github.com/zeit/next.js/tree/canary/examples/with-tailwindcss-emotion
2. Install and go to `/pages/index.js`
3. Add `transition duration-150 ease-in-out` to Header component like below:
```
const Header = styled.div`
  ${tw`font-mono text-sm text-gray-800 hover:text-red-500 transition duration-150 ease-in-out`}
`;
```
4. Run `npm run dev`
5. See error.

## Expected behavior

This shouldn't error out as `transition`, `duration-150`, and `ease-in-out` are all supported by tailwindcss: 
https://tailwindcss.com/docs/transition-property/
https://tailwindcss.com/docs/transition-duration/
https://tailwindcss.com/docs/transition-timing-function/

## Screenshots

<img width="1675" alt="Screen Shot 2020-04-13 at 9 19 56 PM" src="https://user-images.githubusercontent.com/11680080/79186164-c5e35880-7dcd-11ea-876a-4404f0af5354.png">

## System information
- OS: MacOS
- Browser: Chrome
- Version of Next.js: 9.3.4
- Version of Node.js: 10.15.3

## Additional context

This is definitely not a bug with Next.js but wanted to bring attention to it as the official example with tailwindcss + emotion is broken.

**Comment by coktopus** [comment]
https://github.com/zeit/next.js/pull/12027

**Comment by timneutkens** [comment]
Merged that PR, thanks @coktopus!

**Comment by Arthie** [comment]
Hey, @timneutkens this issue is not solved. But can remain closed since its a duplicate of #11288.

PR #12027 used a CSS rule instead of Tailwind transition classes to add a transition.

The tailwind + emotion example is broken because it uses an unmaintained package to handle tailwind classes.

I submitted a PR #11611 that fixes this problem and solves issues: #11288, #11872

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
